app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,31 +11,26 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
         var _instruments = MutableLiveData<ArrayList<Instrument>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
-        call?.enqueue(object: Callback<ArrayList<Instrument>>{
+        call?.enqueue(object : Callback<ArrayList<Instrument>> {
 
             override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {
                 val j = 1 + 1
                 val i = 1 + 1
-
             }
 
-
```

Comment:
Code Review Suggestion 3:
A single blank line should appear "between consecutive members of a class: properties, constructors, functions, nested classes, etc.", according to https://developer.android.com/kotlin/style-guide. There are a couple exceptions to this rule but in this instance, between functions, there should only be a single space seperating them.